platform-sdk/swirlds-common/src/main/java/com/swirlds/common/io/streams/SerializableStreamConstants.java

```diff
@@ -39,7 +39,7 @@ private SerializableStreamConstants() {}
     public static final int VERSION_BYTES = Integer.BYTES;
 
     /**
-     * The current version of the serialization protocol implemented by {@link SerializableDataOutputStream} and
+     * The previous version of the serialization protocol implemented by {@link SerializableDataOutputStream} and
```

Comment:
Was this doc change intentional, or is it left over from a previous changeset? I'm not understanding why it used to be "current", but would now be "previous"

Reply:
Forgot to roll it back. Fixed